### PR TITLE
basic command path finder

### PR DESCRIPTION
**Pull request by letsgogeeky**

### CODE REVIEW

File: src/execution/command_path.c [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+/* ************************************************************************** */
+
+#include <unistd.h>
+#include "minishell/execution/executor.h"
+
+static char	*find_path_in_env(char **envp, char *program)
+{
+	int		idx;
+	char	*path;
+	char	**env_paths;
```
Comment (dendeaisd) [inline src/execution/command_path.c:20]: Can you align these variables so they follow norminette?

File: Makefile [hunk 2]
```diff
@@ -3,10 +3,10 @@
 #                                                         :::      ::::::::    #
 #    Makefile                                           :+:      :+:    :+:    #
 #                                                     +:+ +:+         +:+      #
-#    By: ramymoussa <ramymoussa@student.42.fr>      +#+  +:+       +#+         #
+#    By: ramoussa <ramoussa@student.42.fr>          +#+  +:+       +#+         #
 #                                                 +#+#+#+#+#+   +#+            #
 #    Created: 2023/12/15 20:55:14 by ramoussa          #+#    #+#              #
-#    Updated: 2023/12/19 20:14:50 by ramymoussa       ###   ########.fr        #
+#    Updated: 2023/12/26 20:02:21 by ramoussa         ###   ########.fr        #
```
Comment (dendeaisd) [inline Makefile:9]: Why are they different? xD

**Review by dendeaisd — APPROVED** [review]
lgtm

File: src/execution/command_path.c (hunk 1 above)
Comment (letsgogeeky) [reply src/execution/command_path.c:20]: yeah right i will handle norminette

File: Makefile (hunk 2 above)
Comment (letsgogeeky) [reply Makefile:9]: xDDD

**Pull request merged**